test/test.sql

```diff
@@ -1,7 +1,7 @@
 DROP TABLE person_test;
```

Comment:
@cjbj gotcha. Thanks :-)

There are examples of that pattern in the bottom half of the file and I was planning on following them at some point, along with generally cleaning up the formatting.